.eslintignore

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+**/node_modules/**/*
+**/*bundle.js
+**/dist/*
+**/drop/*
+**/extension/*
```

Comment:
It feels like it would be easy for someone to accidentally create a directory like `/src/unified/platforms/extension/` in the future and miss that it wasn't being linted for a long time. Should most of these `**/` cases be more specific to the root directories?

Reply:
I think perhaps "extension" could specifically be called out at the root, but since we have packages, "node_modules", "drop", and "dist" could occur elsewhere. There is a drop folder under packages/report for example.